src/Jobs/QueuedTenantDatabaseMigrator.php

```diff
@@ -34,5 +42,11 @@ public function handle()
         Artisan::call('tenants:migrate', [
             '--tenants' => [$this->tenant->id],
         ]);
+
+        if ($this->shouldRunSeeder) {
```

Comment:
Could you extract this logic into a separate, queuable `QueuedTenantDatabaseSeeder`?